src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -208,6 +208,14 @@ public static class Parser
             new Regex(@"(\[|\()*\b((featuring|feat.|feat|ft|ft.)\s{1}){1}\s*.*(\]|\))*", RegexOptions.IgnoreCase | RegexOptions.Compiled),
             new Regex(@"(?:\(|\[)(?:[^\(\[]*)(?:version|limited|deluxe|single|clean|album|special|bonus|promo|remastered)(?:[^\)\]]*)(?:\)|\])", RegexOptions.IgnoreCase | RegexOptions.Compiled)
```

Comment:
Can this be removed given the bracket Regex? or do we still need it?

Reply:
These are used for the standand `FindByTitle` so I think we need to keep hold of them.